.redhat-ci.Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dnf install -y \
         python-dbus \
         pylint \
         python-slip-dbus \
-        python-docker-py \
+        python2-docker \
```

Comment:
It could also be possible to pull the package directly from koji. As a workaround before having F26.